### PR TITLE
feat: include partial reserve balances in totals

### LINKED ISSUE #30: Reserve website should include any assets held by the partial reserve
**Description**
During the development of McMento, a new reserve called the [partial reserve](https://explorer.celo.org/mainnet/address/0xea0Cf774eB755fB01FD67fe6B4F93AE4E167afe2) was created to separate funds needed during the rollout phase. To accurately represent the total assets on the website, we need to include collateral and stable assets this new partial reserve holds in the displayed totals.

**Acceptance Criteria**

- [x] Currennt reserve holdings for Celo and non-Celo assets take into account any collateral assets that are held by the partial reserve
- [ ] The outstanding supply of stable assets should include assets that are held in the partial reserve

### PR DESCRIPTION
**Pull request by bayological**
### Description

- Updated to include balance of partial reserve in collateral asset amounts displayed on site
- Outstanding supply of stables includes amounts in partial reserve
- Partial reserve address displayed in list of reserve addresses

### Related issues

- Fixes #30

### CODE REVIEW

**Review by philbow61 — APPROVED** [review]
LGTM 👍

File: src/addresses.config.ts [hunk 1]
```diff
@@ -54,5 +54,7 @@ export function generateLink(token: Tokens, address: string) {
     case "cUSD in Curve Pool":
     case "USDC in Curve Pool":
       return `https://explorer.celo.org/mainnet/address/${address}/tokens#address-tabs`
+    case "Partial Reserve":
+      return `https://celoscan.io/address/${address}`
```
Comment (jmrossy) [inline src/addresses.config.ts:58]: Recommend matching other links which use blockscout or updating them

**Review by jmrossy — APPROVED** [review]

File: src/providers/Celo.ts [hunk 2]
```diff
@@ -226,6 +236,21 @@ export async function getMultisigUSDC() {
   return providerOk(usdcWormhole.value + usdcAxelar.value, Providers.celoNode)
 }
 
+export async function getPartialReserveCUSD() {
+  return getERC20Balance(CUSD_ADDRESS, PARTIAL_RESERVE_ADDRESS)
+}
```
Comment (bowd) [inline src/providers/Celo.ts:241]: I don't think we need this.

File: src/service/stables.ts [hunk 3]
```diff
@@ -67,12 +79,16 @@ export default async function stables(): Promise<TokenModel[]> {
...
+    // This adjusts the total supply to account for the reserve owned CUSD that have already been minted
     if (tokenData.symbol === StableToken.cUSD) {
       value -= curveCUSDAmount * prices.value[tokenData.iso4217]
       units -= curveCUSDAmount
 
       value -= multisigCUSDAmount * prices.value[tokenData.iso4217]
       units -= multisigCUSDAmount
+
+      value -= partialReserveCUSDAmount * prices.value[tokenData.iso4217]
+      units -= partialReserveCUSDAmount
```
Comment (bowd) [inline src/service/stables.ts:91]: Nor this – basically the partial reserve is equivalent to the normal reserve contract, it will never hold cUSD.

**Review by bowd — CHANGES_REQUESTED** [review]
Unless I'm missing something, the partial reserve will never hold cUSD - just like the normal reserve doesn't and there's no accounting for that in the logic.

It should only affect the tally of collateral assets in the system.

**Review by bowd — APPROVED** [review]

**Pull request merged**